pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -8,7 +8,6 @@ import (
 	"path/filepath"
 
 	"github.com/YourFin/binappend"
-	"github.com/kardianos/osext"
```

Comment:
@cfergeau shouldn't this also change the go module data, you might need to run `go mod tidy` after this change and if we are not using `osext` anywhere else then it should be removed from the modules.

Reply:
`go mod tidy` is blocked by https://github.com/code-ready/machine/issues/28

Reply:
@cfergeau I am using golang-1.12.13 on osx and didn't get any issue when running the `go mod tidy`

```
$ go mod why github.com/kardianos/osext
warning: ignoring symlink /Users/prkumar/work/github/crc/docs/source/getting-started/common-content
go: finding github.com/code-ready/machine/libmachine/provision latest
go: finding github.com/code-ready/machine/commands/mcndirs latest
go: finding github.com/code-ready/machine/commands latest
go: finding github.com/code-ready/machine/libmachine latest
go: finding github.com/code-ready/machine latest
# github.com/kardianos/osext
github.com/code-ready/crc/pkg/os
github.com/kardianos/osext

$ git diff
diff --git a/pkg/os/util.go b/pkg/os/util.go
index 3c7ab82..b7cac46 100644
--- a/pkg/os/util.go
+++ b/pkg/os/util.go
@@ -11,7 +11,6 @@ import (
        "strings"
 
        "github.com/code-ready/crc/pkg/crc/logging"
-       "github.com/kardianos/osext"
 )
 
 const (
@@ -55,7 +54,7 @@ func ReplaceEnv(variables []string, varName string, value string) []string {
 }
 
 func CurrentExecutable() (string, error) {
-       currentExec, err := osext.Executable()
+       currentExec, err := os.Executable()
        if err != nil {
                return "", err
        }

$ go mod why github.com/kardianos/osext
warning: ignoring symlink /Users/prkumar/work/github/crc/docs/source/getting-started/common-content
go: finding github.com/code-ready/machine/libmachine/provision latest
go: finding github.com/code-ready/machine/commands/mcndirs latest
go: finding github.com/code-ready/machine/libmachine latest
go: finding github.com/code-ready/machine/commands latest
go: finding github.com/code-ready/machine latest
# github.com/kardianos/osext
(main module does not need package github.com/kardianos/osext)

 $ go mod tidy
warning: ignoring symlink /Users/prkumar/work/github/crc/docs/source/getting-started/common-content
go: finding github.com/code-ready/machine/libmachine/provision latest
go: finding github.com/code-ready/machine/commands/mcndirs latest
go: finding github.com/code-ready/machine/commands latest
go: finding github.com/code-ready/machine/libmachine latest
go: finding github.com/code-ready/machine latest

$ git diff
diff --git a/go.mod b/go.mod
index e32534e..d43025d 100644
--- a/go.mod
+++ b/go.mod
@@ -11,7 +11,6 @@ require (
        github.com/code-ready/machine v0.0.0-20190904103148-0fde37b5d95b
        github.com/docker/go-units v0.4.0
        github.com/inconshreveable/mousetrap v1.0.0 // indirect
-       github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
        github.com/mattn/go-colorable v0.0.9
        github.com/pborman/uuid v1.2.0
        github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
diff --git a/pkg/os/util.go b/pkg/os/util.go
index 3c7ab82..b7cac46 100644
--- a/pkg/os/util.go
+++ b/pkg/os/util.go
@@ -11,7 +11,6 @@ import (
        "strings"
 
        "github.com/code-ready/crc/pkg/crc/logging"
-       "github.com/kardianos/osext"
 )
 
 const (
@@ -55,7 +54,7 @@ func ReplaceEnv(variables []string, varName string, value string) []string {
 }
 
 func CurrentExecutable() (string, error) {
-       currentExec, err := osext.Executable()
+       currentExec, err := os.Executable()
        if err != nil {
                return "", err
        }
```